VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -44,6 +50,21 @@ public void setAppNameContext() {
             this.properties.put(Keys.CONTEXT, getAppNameContext());
         }
 
+        public void setCustomDimensions() {
```

Comment:
> The whole purpose of the function is to set custom dimensions, so, why name it in any other way?

I guess you mean "dimensions" to mean generic "properties" here, while I was thinking of the technical GA event? In which case, this should be renamed to something less ambiguous in any case…

> Having a function like such, will make it easy for us to add more dimensions in the future, as we'll have a whole function for it and won't have to search the code block/lines where to add them.

OK, but at least make it private in that case.